docs/features/banner-designer.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Banner Designer"
+description: "Customize the consent banner & preferences modal to match your branding."
+---
+
+<img src="/images/consent-banner/banner-designer.png" alt="Banner Designer" />
+
+## Geo Targeting
+
+Banners can be set up to show worldwide or only in certain countries or US states.
+
+Multiple banners can be set up in this way to match the user's country and their regulations.
+
+<img src="/images/consent-banner/geo-targeting.png" alt="Geo Targeting" />
+
+## Force Consent
+
+The force consent option can be toggled to increase consent rates.
+
+When this option is enabled, the user is forced to accept/deny consent before interacting with the website.
+
+<img src="/images/consent-banner/force-consent.png" alt="Force Consent" />
+
+## Layouts
+
+The layout, position and button placement of banners can be customized to fit your website's style.
+<img src="/images/consent-banner/banner-layout.png" alt="Banner Layout" />
+
+Similarly, the layout of the preferences modal can also be customized.
+<img src="/images/consent-banner/preferences-layout.png" alt="Preferences Layout" />
+
+## Banner Buttons
+
+The weights, position and text of banner buttons can be customized.
+
+<Note>
+  Some regulations like GDPR require banner buttons to be equally weighted.
+  Manage preferences button must be enabled to give fine granular control to the users.
+  Some regulations might require specific text for buttons.
+</Note>
+
+<img src="/images/consent-banner/banner-buttons.png" alt="Banner Buttons" />
+
+## Privacy Trigger
+
+The privacy trigger that allows users to update their consent preferences at any time, can be enabled/disabled and its position can be customized.
+
+<Note>
+  Some regulations will require this feature to be enabled.
+</Note>
```

Comment:
```suggestion
<Note>
  Most regulations mandate that users must be able to update their consent preferences after giving their initial consent. You can enable this using our Privacy Trigger or by adding your own button or link to open the preferences.
</Note>
```

link the you can add your own button to the other page where we talk about the syntax to open the modal with the data attribute